### DISCUSSION TITLE
Does Cascade solution still in the plan?

### DISCUSSION THREAD

**Discussion by herryfan** [OpenSight/janus-cloud#14] [Q&A]
Hi,

Does cascade solution still in the plan?

Thanks!

**Comment by gushenjie** [comment]
Does cascade solution still in the plan?

**Comment by gushenjie** [comment]
![image](https://github.com/user-attachments/assets/fb471de5-e141-44c2-b14c-41f7763d1e42)
I have proxy, sentinel, and janus installed on one server, and janus and sentinel installed on the other. I turned on cascade to true, and I found that multi-stream janus did not report an error, but now there is no screen.
